internal/redfishwrapper/power.go

```diff
@@ -221,3 +220,19 @@ func (c *Client) SystemForceOff(ctx context.Context) (ok bool, err error) {
 
 	return true, nil
 }
+
+// SendNMI tells the BMC to issue an NMI to the device
+func (c *Client) SendNMI(_ context.Context) error {
```

Comment:
before invoking `Systems()`, add in a session check - so we don't end up with nil panics because the redfish session wasn't initialised or has expired.
```
	if err := c.SessionActive(); err != nil {
		return false, errors.Wrap(bmclibErrs.ErrNotAuthenticated, err.Error())
	}
```

Reply:
Good catch, done.